app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
+      session[:user_id] = user.id
+      flash[:success] = 'You have successfully logged in'
+      redirect_to root_path
+    else
+      flash.now[:danger] = 'There is something wrong with your login information'
+      render 'new'
+    end
+  end
+
+  def destroy
+    session[:user_id] = nil
```

Comment:
Let's set cookies to nil each time the user logs out.
We can use a helper method for this that looks similar to.
```ruby
def sign_out
  @current_user = nil
  cookies.delete(:gen_token)
end
```